sklearn/tests/test_multioutput.py

```diff
@@ -648,3 +653,116 @@ def test_classifier_chain_tuple_invalid_order():
 
     with pytest.raises(ValueError, match='invalid order'):
         chain.fit(X, y)
+
+
+def test_classifier_chain_verbose(capfd):
+    X, y = make_multilabel_classification(
+        n_samples=100,
+        n_features=5,
+        n_classes=3,
+        n_labels=3,
+        random_state=0)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+
+    # default verbose (False)
+    base_clf = DecisionTreeClassifier()
+    chain = ClassifierChain(
+        base_clf,
+        order='random',
+        random_state=0)
+    chain.fit(X_train, y_train)
+    out, err = capfd.readouterr()
+    assert out == err == ''
+
+    # verbose explicitly False
+    base_clf = DecisionTreeClassifier()
+    chain = ClassifierChain(
+        base_clf,
+        order='random',
+        random_state=0,
+        verbose=False)
+    chain.fit(X_train, y_train)
+    out, err = capfd.readouterr()
+    assert out == err == ''
+
+    # verbose True
+    chain = ClassifierChain(
+        base_clf,
+        order='random',
+        random_state=0,
+        verbose=True)
+    chain.fit(X_train, y_train)
+    out, err = capfd.readouterr()
+    assert out != '' and err == ''
+
+
+def test_regressor_chain_verbose(capfd):
+    X, y = make_regression(n_samples=125, n_targets=3, random_state=0)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
+
+    # default verbose (False)
+    base_reg = LinearRegression()
+    chain = RegressorChain(
+        base_reg,
+        order=[1, 0, 2],
+        random_state=0)
+    chain.fit(X_train, y_train)
+    chain.fit(X_train, y_train)
+    out, err = capfd.readouterr()
+    assert out == err == ''
+
+    # verbose explicitly False
+    base_reg = LinearRegression()
+    chain = RegressorChain(
+        base_reg,
+        order=[1, 0, 2],
+        random_state=0,
+        verbose=False)
+    chain.fit(X_train, y_train)
+    chain.fit(X_train, y_train)
+    out, err = capfd.readouterr()
+    assert out == err == ''
+
+    # verbose True
+    base_reg = LinearRegression()
+    chain = RegressorChain(
+        base_reg,
+        order=[1, 0, 2],
+        random_state=0,
+        verbose=True)
+    chain.fit(X_train, y_train)
+    chain.fit(X_train, y_train)
+    out, err = capfd.readouterr()
+    assert out != '' and err == ''
+
+
+def test_multi_label_y():
```

Comment:
Same as above: how this change is related to the verbose issue?

Reply:
We should open a new PR if this is a bug.

Reply:
I'll open a new PR for it